src/ProjectSystem/Microsoft.VisualStudio.Editors/project.json

```diff
@@ -3,5 +3,9 @@
     "net46": {}
   },
   "dependencies": {
+    "Microsoft.VisualStudio.LanguageServices": {
+      "version": "1.2.0",
+      "suppressParent": "all"
```

Comment:
suppressParent causes the binaries to not get dropped in the output folder? If so, that's a horrible name.